runtime/queries/fennel/highlights.scm

```diff
@@ -0,0 +1,196 @@
+; Most primitive nodes
+(shebang) @keyword.directive
+
+(comment) @comment @spell
```

Comment:
```suggestion
(comment) @comment
```

Neovim includes spellchecking queries in the highlights. When we introduce tree-sitter based spellchecking we will keep the queries in a different file so we can remove all `@spell` captures